cerberus-audit-logger-athena/src/test/java/com/nike/cerberus/audit/logger/service/AthenaServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package com.nike.cerberus.audit.logger.service;
+
+import static junit.framework.Assert.assertTrue;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+import com.nike.cerberus.audit.logger.AthenaClientFactory;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+public class AthenaServiceTest {
+
+  AthenaClientFactory athenaClientFactory = new AthenaClientFactory();
```

Comment:
How about putting initialization code in a constructor or the @Before method? What is the scope of this member variable (public, private, etc.)?

Reply:
i will  modify this